rest.go

```diff
@@ -57,26 +88,23 @@ func (erre *EncodingError) Error() string {
 	return "leaseweb: encoding JSON request body failed (" + erre.Err.Error() + ")"
 }
 
+// Deprecated: Use SetDefaultAccount instead
```

Comment:
Because we are still on version 0.y.z, there is no need to manage deprecations yet and those functions can just be removed. It is quite standard for a version 0.y.z to not be considered stable and have constant breaking changes.